repldex/backend/templates/entry.html

```diff
@@ -8,87 +8,60 @@
 		<meta property="og:image" content="{{ image }}">
 	{% endif %}
 
-	<script>
+<script>
 document.addEventListener('DOMContentLoaded', () => {
```

Comment:
You converted all my beautiful tabs into ugly spaces >:(

Reply:
no shush